netmaster/master/endpoint.go

```diff
@@ -118,6 +118,13 @@ func CreateEndpoint(stateDriver core.StateDriver, nwCfg *mastercfg.CfgNetworkSta
 	epCfg.ServiceName = ep.ServiceName
 	epCfg.EPCommonName = epReq.EPCommonName
 
+	// In ACI mode, if a pod does not have a group label, we will assume "default-group"
+	isAci, _ := IsAciConfigured()
+
+	if isAci && (len(epCfg.ServiceName) == 0) {
+		epCfg.ServiceName = "default-group"
```

Comment:
Please add a log.Infof here with a message we are overriding withe default-group.